admin-dev/themes/default/scss/partials/_buttons.scss

```diff
@@ -1,15 +1,81 @@
 .btn {
+  font-weight: 600;
+
   &.btn-default {
+    &,
+    i {
+      color: $medium-gray;
+      transition: 0.25s ease-out;
+    }
+
     &:hover {
-      background-color: $primary-light-color;
-      border-color: $primary-light-color;
+      color: #fff;
```

Comment:
Why not using a new variable like $btn-color?

Reply:
It could be a thing yes

Reply:
Ping @NeOMakinG